src/forward_link.py

```diff
@@ -0,0 +1,105 @@
+from typing import List
```

Comment:
Can you double check that typing is added to the requirments.txt

Reply:
This by default  comes with python